buildSrc/src/main/kotlin/pklKotlinLibrary.gradle.kts

```diff
@@ -38,3 +41,7 @@ dependencies {
 tasks.compileKotlin {
   enabled = true // disabled by pklJavaLibrary
 }
+
+tasks.withType<KotlinJvmCompile>().configureEach {
+  compilerOptions { jvmTarget = JvmTarget.JVM_17 }
+}
```

Comment:
this will rebase away when #876 is in; plus, this block is better customized by #920